### PR TITLE
maguro: Use USE_MAGURO_LIBCAMERA to conditional select the camera interf...

### PR DESCRIPTION
**Pull request by kanru**
...ace

### CODE REVIEW

File: glue/gonk/device/toro/maguro/BoardConfig.mk [hunk 1]
```diff
@@ -60,6 +60,7 @@ QCOM_TARGET_PRODUCT := msm7627a_sku1
 # Enable temporary stub methods that permit the newer libcamera.so blob
 # to link with the older source
 TARGET_LIBCAMERA_BLOB_SYM_MISMATCH := true
+TARGET_GLOBAL_CPPFLAGS += -DUSE_MAGURO_LIBCAMERA
```
Comment (joneschrisg) [inline glue/gonk/device/toro/maguro/BoardConfig.mk:63]: For future reference, this isn't a maguro-specific libcamera interface, it just happens to be a newer camera HAL interface that the blobs in our current marguro firmware rely on.  So it'd be better to name things like this to capture the differing SW features than for the hardware name in the future.

But I don't care that much :).

**Comment by joneschrisg** [comment]
This should be pulled in atomically with https://github.com/cgjones/android-frameworks-base/pull/5 .

**Comment by joneschrisg** [comment]
@shianyow did you mean the three lines starting with

```
// TODO(mvines): Hardcode camera mode to '1' (2D) until B2G moves up to a newer
```

?  If so I think you're right.

**Comment by shianyow** [comment]
Yes, the three lines.
 2012-1-21 U8:45  "Chris Jones" <
reply@reply.github.com>
gDG

**Comment by andreasgal** [comment]
This is really old. Still needed?

**Comment by kanru** [comment]
no.

**Pull request closed without merging**